lib/views/register.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:unifit/services/auth_service.dart';
+import 'package:unifit/utils/constants.dart';
+import 'package:unifit/widgets/gradient_button.dart';
+
+class SignUpView extends StatefulWidget {
+  const SignUpView({Key? key}) : super(key: key);
+
+  @override
+  _SignUpViewState createState() => _SignUpViewState();
+}
+
+class _SignUpViewState extends State<SignUpView> {
+  final TextEditingController _usernameController = TextEditingController();
+  final TextEditingController _courseController = TextEditingController();
+  final TextEditingController _emailController = TextEditingController();
+  final TextEditingController _passwordController = TextEditingController();
+  final TextEditingController _repasswordController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: AppColors.MAIN_BLUE,
+      body: Padding(
+        padding: const EdgeInsets.all(30.0),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Image.asset('lib/assets/layers/unifit_logo.png', width: 320.0),
+            TextField(
```

Comment:
Qualquer widget que defina width e heigth devem ser sempre relativos aos valores do aparelho. A altura e largura das telas podem ser obtidas dessa maneira:
double screenHeight = MediaQuery.of(context).size.height;
double screenWidth = MediaQuery.of(context).size.width;
Em seguida dividir a altura/largura de acordo com o valor desejado:
320 / screenWidth 
O double obtido deverá ser utilizado como multiplicador da largura/altura da tela, por exemplo:
 Image.asset('lib/assets/layers/unifit_logo.png', width: doubleObtidoAtravesDoCalculo * screenWidth)